installer/resources/lambda_submit/function.py

```diff
@@ -15,13 +15,65 @@
 from resources.pacbot_app.utils import  get_azure_tenants,  get_gcp_project_ids, get_aws_account_details
 import json
 from core.config import Settings
+from resources.iam.eventbridge_role import EventBridgePolicyRole
 
+class EventTrigger(CloudWatchEventRuleResource):
+    name = "Trigger-event"
+    event_pattern = {
+        "$or": [
+            {
+                "detail-type": ["Batch Job State Change"],
+                "source": ["aws.batch"],
+                "detail": {
+                    "status": ["SUCCEEDED"],
+                    "jobName": ["AWS-Data-Collector-job"],
```

Comment:
Please use the same case format for all names. This either need to change to all lower case or other need to change camel case.